Cargo.toml

```
@@ -21,23 +21,37 @@ exclude = [".github/", "assets/"]
 # ---------------------------------------------------------------------------- #
 
 [dependencies]
-bevy = { version = "0.16.0", default-features = false, features = [
+bevy = { version = "0.17", default-features = false, features = [
```

```suggestion
bevy = { version = "0.17.0", default-features = false, features = [
```

I prefer having the full version

---

Sure.